content/en/blog/2024/otel-collector-anti-patterns/index.md

```diff
@@ -0,0 +1,184 @@
+[Collector distributions](/ecosystem/distributions/). These are OTel Collector
+distributions that are curated to Collector components that are specific to that
+vendor. They may be a combination of custom, vendor-developed components, and
+curated Collector Contrib components. Using vendor-specific distributions
+ensures that you are using just the Collector components that you need, again
+reducing overall bloat.
+
+_Bottom line:_ Using the right distribution reduces bloat and allows you to
+include only the Collector components that you need.
+
+### 4- Not updating your Collectors
+
+This one’s short and sweet. Keeping software up-to-date is important, and the
+Collector is no different! By regularly updating the Collector, it allows you to
+stay up-to-date with the latest version so that you can take advantage of new
+features, bug fixes, performance improvements, and security fixes.
+
+### 5- Not using the OpenTelemetry Collector where appropriate
+
+OpenTelemetry allows you to send telemetry signals from your application to an
+observability backend in one of two ways:
+
+- [Directly from the application](/docs/collector/deployment/no-collector/)
+- [Via the OpenTelemetry Collector](/docs/collector/)
+
+Sending telemetry data “direct from application” for non-production systems is
+all well and good if you’re getting started with OpenTelemetry, but it is
+neither suited nor recommended to use this approach for production systems.
+Instead, the
+[OpenTelemetry docs recommend using the OpenTelemetry Collector](/docs/collector/#when-to-use-a-collector).
+How come?
+
+[Per the OTel Docs](/docs/collector/#when-to-use-a-collector), the Collector
+“allows your service to offload data quickly and the collector can take care of
+additional handling like retries, batching, encryption or even sensitive data
+filtering.”
+
+Check out some additional Collector benefits:
+
+- **Collectors can enhance the quality of the telemetry emitted by an
+  application while also minimizing costs.** For example: sampling spans to
+  reduce costs, enriching telemetry with extra metadata, and generating new
+  telemetry, such as metrics derived from spans.
+- **Using a Collector to ingest telemetry data makes it easy to change to a new
+  backend or export the data in a different format.** If we want to change how
+  telemetry is being processed or exported, that change happens in one place
+  (the Collector!), as opposed to making the same change for multiple
+  applications in your organization.
+- **Collectors allow you to receive data of various formats and translate to the
+  desired format for export.** This can be very handy when transitioning from
+  some other telemetry solution to OTel.
+- **Collectors allow you to ingest non-application telemetry.** This includes
+  logs and non-app metrics from infrastructure like Azure, Prometheus, and
+  Cloudwatch.
+
+That being said, there are some use-cases where folks don't want or can't use a
```

Comment:
@jpkrohling added some clarification here, and also updated the section heading to better describe this

Reply:
Sounds good now!